tests/unit/local/apigw/test_local_apigw_service.py

```diff
@@ -386,14 +388,14 @@ def setUp(self):
         self.request_mock.method = "GET"
         self.request_mock.remote_addr = "190.0.0.0"
         self.request_mock.get_data.return_value = b"DATA!!!!"
-        query_param_args_mock = Mock()
-        query_param_args_mock.lists.return_value = {"query": ["params"]}.items()
+        query_param_args_mock = ImmutableMultiDict(MultiDict({"query": ["params"]}))
```

Comment:
Let's not change this. Let's keep the mocks.

Reply:
Reverting it.